### ISSUE TITLE
Table line deletion breaks the document and moves permanently the cells comments

### ISSUE THREAD

**Issue by jlemangarin** [ONLYOFFICE/DesktopEditors#1717]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Windows 10 / 8.1 / 7

### Version information

8.2.0.143

### Expected Behavior

Only the selected line is removed and the cell merging stays (From Excel behavior)

![image](https://github.com/user-attachments/assets/6930ef31-2cc0-436f-b97a-e0e58264d978)

### Actual Behavior

- The entire table is broken and comments are moved
- CTRL+Z makes the table reappear, but the comments are moved a few lines below

### Reproduction Steps

1. Open [the attached document](https://github.com/user-attachments/files/17617970/test_bug.OO.xlsx)
2. Select entire line number #5 that shows "ERROR" on column B (by clicking on the line number), right click, delete
3. Table is now broken
4. CTRL+Z
5. Table is now restored, but some comments in the table have moved under it (F27 and I30)

### Additional information

Win 11 last OO update, reproduced on another document

**Comment by devappsrdsco** [comment]
Yes, I have noticed similar behavior when the document is saved and reopened. Upon reopening, some rows are shifted down. I am rolling back to version 8.1.1 and testing with the broken file.

**Comment by Dmitry-Kov** [comment]
Hi @jlemangarin,

I created ticket #71499 with your issue. When some fixes will be done we'll text you here in this issue.
Thank you for the report!

Problem can be reproduced in DocumentServer.
